src/hacbs/components/Commits/CommitsListRow.tsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { GithubIcon } from '@patternfly/react-icons/dist/js/icons/github-icon';
+import ActionMenu from '../../../shared/components/action-menu/ActionMenu';
+import ExternalLink from '../../../shared/components/links/ExternalLink';
+import { RowFunctionArgs, TableData } from '../../../shared/components/table';
+import { Timestamp } from '../../../shared/components/timestamp/Timestamp';
+import { Commit } from '../../types';
+import { useCommitActions } from './commit-actions';
+import { commitsTableColumnClasses } from './CommitsListHeader';
+
+const CommitsListRow: React.FC<RowFunctionArgs<Commit>> = ({ obj }) => {
+  const actions = useCommitActions(obj);
+  return (
+    <>
+      <TableData className={commitsTableColumnClasses.name}>
+        <Link to={`/app-studio/commit?name=${obj.sha}`} title={obj.sha} className="pf-u-mr-sm">
+          {obj.sha.length > 14 ? `${obj.sha.slice(0, 14)}...` : obj.sha}
+        </Link>
+        {obj.shaURL && (
+          <ExternalLink href={obj.shaURL}>
+            <GithubIcon />
+          </ExternalLink>
+        )}
+      </TableData>
+      <TableData className={commitsTableColumnClasses.branch}>
+        {obj.gitProvider === 'github' && obj.repoOrg && obj.branch ? (
+          <ExternalLink
+            href={`https://github.com/${obj.repoOrg}/tree/${obj.branch}`}
+            text={`[${obj.branch}]`}
+          />
+        ) : (
```

Comment:
Clicking on the branch value takes to 404 page not found and also this will work only for `github` provider, PAC supports gitlab, bitbucket urls as well.